Tests/CocoaPodsTest/PodInstall/PodInstall/ViewController.swift

```diff
@@ -6,7 +6,11 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+
+        // Test that classes are accessible
+        // and prevent comiler optimizations for unused imports.
+        _ = PassiveLocationManager()
+        _ = NavigationMapView(frame: .zero)
```

Comment:
I'm curious where this change comes from?

Reply:
I've added a comment for this. If we want to test integration here, it's better to use some classes from the SDK to prevent compiler optimizations for unused imports.